modules/gapi/include/opencv2/gapi/infer.hpp

```diff
@@ -121,6 +121,47 @@ struct GInferBase {
     }
 };
 
+struct InOutInfo
+{
+    std::vector<std::string> in_names;
+    std::vector<std::string> out_names;
+};
+
+class GInferInputs
+{
+public:
+    cv::GMat& operator[](const std::string& name) { return in_blobs[name]; }
+    const std::unordered_map<std::string, cv::GMat>& getBlobs() const { return in_blobs; }
+
+private:
+    std::unordered_map<std::string, cv::GMat> in_blobs;
+};
```

Comment:
Since this is not a template, please move the method implementations into the `.cpp`.

Reply:
Fixed